### ISSUE TITLE
[BUG] input v0 converting to input v2 bug

### ISSUE THREAD

**Issue by jxxiaoshaoye** [deepmodeling/deepmd-kit#919]
**attribute error when converting input versions**
When I run input.json v0 under environment deepmd-kit v2.0.0.b3, it would first convert it into input_v2_compat.json. However, it still have key error like `decay_rate`/ `n_neuron` is not allowed in strict mode. 
I attached my input file below:
```
{
    "model": {
        "descriptor": {
            "type": "se_a",
            "sel": [
                500
            ],
            "rcut_smth": 2.0,
            "rcut": 9.0,
            "neuron": [
                25,
                50,
                100
            ],
            "resnet_dt": false,
            "axis_neuron": 12,
            "seed": 1084011120
        },
        "fitting_net": {
            "n_neuron": [
                240,
                240,
                240
            ],
            "resnet_dt": true,
            "coord_norm": true,
            "type_fitting_net": false,
            "seed": 3151311092
        },
        "type_map": [
            "Re"
        ]
    },
    "learning_rate": {
        "type": "exp",
        "start_lr": 0.0001,
        "decay_steps": 25000,
        "decay_rate": 0.95
    },
    "loss": {
        "start_pref_e": 100,
        "limit_pref_e": 100,
        "start_pref_f": 1,
        "limit_pref_f": 1,
        "start_pref_v": 2.0,
        "limit_pref_v": 2.0
    },
    "training": {
        "systems": ['systems'],
        "set_prefix": "set",
        "stop_batch": 5000000,
        "batch_size": "auto",
        "seed": 813129136,
        "_comment": "that's all",
        "disp_file": "lcurve.out",
        "disp_freq": 2000,
        "numb_test": 2,
        "save_freq": 2000,
        "save_ckpt": "model.ckpt",
        "load_ckpt": "model.ckpt",
        "disp_training": true,
        "time_training": true,
        "profiling": false,
        "profiling_file": "timeline.json"
    }
}
```
The error log is like
![image](https://user-images.githubusercontent.com/53417696/128145530-181166de-baf5-4b1a-9b1d-794504eb3067.png)

**Comment by njzjz** [comment]
`load_skpt` is also deprecated. See #907.

**Comment by jxxiaoshaoye** [comment]
yeah, but `n_neuron` is not automatically converted into `neuron`? in other words, when I'm using v0 input, I should change the input file first by myself?

**Comment by amcadmus** [comment]
> yeah, but `n_neuron` is not automatically converted into `neuron`? in other words, when I'm using v0 input, I should change the input file first by myself?

What is the version of your `dargs`? Have you updated it to the latest release 0.2.6?

**Comment by jxxiaoshaoye** [comment]
it's the latest version 0.2.6

**Comment by njzjz** [comment]
@jxxiaoshaoye As I commented in https://github.com/deepmodeling/deepmd-kit/issues/846#issuecomment-894863451, the correct key is `neuron` but not `n_neuron` in v1. If you used `n_neuron` in v1, `neuron` will fallback to the default value `[120,120,120`], and you may need to check the previous model.

**Comment by njzjz** [comment]
- `n_neuron` was fixed in #932 and #933, but it does not work from v1.3.0 to v1.3.3
- `decay_rate` and `load_ckpt` should be processed in `compat.py`. I think `load_ckpt` has already deprecated in v1, but we never remove it in examples and documents?
- `coord_norm` and `type_fitting_net` are deprecated keys in v0? But we never processed it in `compat.py`

**Comment by amcadmus** [comment]
> 
> 
>     * `n_neuron` was fixed in [add aliases to Arguments #932](https://github.com/deepmodeling/deepmd-kit/pull/932) and [add aliases to Arguments #933](https://github.com/deepmodeling/deepmd-kit/pull/933), but it does not work from v1.3.0 to v1.3.3
> 
>     * `decay_rate` and `load_ckpt` should be processed in `compat.py`. I think `load_ckpt` has already deprecated in v1, but we never remove it in examples and documents?
> 
>     * `coord_norm` and `type_fitting_net` are keys in v0 that are deprecated in v1? But we never processed it in `compat.py`

I will rm `load_ckpt` from examples.

`coord_norm` and `type_fitting_net` are deprecated keys
